File: POO/Python/uberBan.py
from car import Car

class UberBan(Car):
    typeCarAccepted = []
    seatMaterial= []

    def __init__(self, license, driver, typeCarAccepted, seatMaterial):
        super.__init__(license, driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatMaterial = seatMaterial

File: POO/Java/UberBan.java
package Java;
import java.util.ArrayList;
import java.util.Map;

class UberBan extends Car
{
   Map<String, Map<String, Integer>> typeCarAccepted;
   ArrayList<String> seatMaterial;

    public UberBan(String license, Account driver, 
    Map<String, Map<String, Integer>> typeCarAccepted, 
    ArrayList<String> seatMaterial)
    {
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatMaterial = seatMaterial;
    }

}